importer/api/__init__.py

```diff
@@ -193,11 +199,22 @@ def import_children(course, element, parent, parent_dpath):
                                 course__id=resource.course_id,
                                 asset=course_asset_basepath(course, path),
                             )
-                            resource.static_assets.add(asset)
+                            static_assets_to_save.add((resource, asset))
                         except StaticAsset.DoesNotExist:
                             continue
                 except AttributeError:
                     continue  # not a string
+    # bulk insert of static assets
```

Comment:
Can you update the comment to explain about the m2m_changed signal we're trying to avoid?